python/strings_udf/strings_udf/_typing.py

```diff
@@ -120,6 +119,18 @@ def prepare_args(self, ty, val, **kwargs):
 str_view_arg_handler = StrViewArgHandler()
 
 
+# for use in testing only
+def sv_to_udf_str(sv):
```

Comment:
If this is only for testing, why not define it in the tests module?

Reply:
I assume it's because of the registration below. I think that probably belongs in the test too, though.

Reply:
The problem is that we need to call this function both in `test_string_udfs.py` (inside the `strings_udf` package) _and_ inside `test_udf_masked_ops.py` in cuDF. If I defined this inside `test_string_udfs.py`, I would need to make that testing file a module that `cudf` could import from, or redefine it inside of `cudf`. I wasn't sure what the lesser of all the evils is here.

Reply:
I think maybe having a separate module as part of `strings_udf` named `_testing` or something similar which contains this code and may be imported but isn't user facing could be a good middle ground here. Let me set it up that way and see how it looks.

Reply:
Would this problem go away when #12669 is merged? If so, maybe we wait to finish that up?